core/templates/pages/feedback-updates-page/feedback-updates-page-root.component.spec.ts

```diff
@@ -0,0 +1,159 @@
+          useClass: MockTranslateService
+        }
+      ],
+      schemas: [NO_ERRORS_SCHEMA]
+    }).compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(FeedbackUpdatesPageRootComponent);
+    component = fixture.componentInstance;
+    pageHeadService = TestBed.inject(PageHeadService);
+    loaderService = TestBed.inject(LoaderService);
+    accessValidationBackendApiService = TestBed.inject(
+      AccessValidationBackendApiService);
+    translateService = TestBed.inject(TranslateService);
+  });
+
+  it('should successfully instantiate the component',
+    () => {
+      expect(component).toBeDefined();
+    });
+
+  it('should initialize and show page when access is valid', fakeAsync(() => {
+    spyOn(accessValidationBackendApiService, 'validateCanManageOwnAccount')
+      .and.returnValue(Promise.resolve());
+    spyOn(loaderService, 'showLoadingScreen');
+    spyOn(loaderService, 'hideLoadingScreen');
+
+    component.ngOnInit();
+    tick();
+
+    expect(loaderService.showLoadingScreen).toHaveBeenCalled();
+    expect(accessValidationBackendApiService.validateCanManageOwnAccount)
+      .toHaveBeenCalled();
+    expect(component.pageIsShown).toBeTrue();
+    expect(component.errorPageIsShown).toBeFalse();
+    expect(loaderService.hideLoadingScreen).toHaveBeenCalled();
+  }));
+
+  it('should initialize and show error page when server respond with error',
+    fakeAsync(() => {
+      spyOn(accessValidationBackendApiService, 'validateCanManageOwnAccount')
+        .and.returnValue(Promise.reject());
+      spyOn(loaderService, 'showLoadingScreen');
+      spyOn(loaderService, 'hideLoadingScreen');
+
+      component.ngOnInit();
+      tick();
+
+      expect(loaderService.showLoadingScreen).toHaveBeenCalled();
+      expect(accessValidationBackendApiService.validateCanManageOwnAccount)
+        .toHaveBeenCalled();
+      expect(component.pageIsShown).toBeFalse();
+      expect(component.errorPageIsShown).toBeTrue();
+      expect(loaderService.hideLoadingScreen).toHaveBeenCalled();
+    }));
+
+  it('should initialize and subscribe to onLangChange', fakeAsync(() => {
+    spyOn(accessValidationBackendApiService, 'validateCanManageOwnAccount')
+      .and.returnValue(Promise.resolve());
+    spyOn(component.directiveSubscriptions, 'add');
+    spyOn(translateService.onLangChange, 'subscribe');
+
+    component.ngOnInit();
+    tick();
+
+    expect(component.directiveSubscriptions.add).toHaveBeenCalled();
+    expect(translateService.onLangChange.subscribe).toHaveBeenCalled();
+  }));
+
+  it('should update page title whenever the language changes', () => {
+    spyOn(accessValidationBackendApiService, 'validateCanManageOwnAccount')
+      .and.returnValue(Promise.resolve());
+    component.ngOnInit();
+    spyOn(component, 'setPageTitleAndMetaTags');
+
+    translateService.onLangChange.emit();
+
+    expect(component.setPageTitleAndMetaTags).toHaveBeenCalled();
```

Comment:
Can you add a check for the title too?

Reply:
I think the test below checks the page title.
Thanks!